### PR TITLE
Frist commit for currency converter

### PR DESCRIPTION
**Pull request by tavisca-dchougule**
@ytokas  frist commit of currency converter code. please review the code.

### CODE REVIEW

**Comment by tavisca-dchougule** [comment]
@ytokas  modified currency converter. please review.

**Comment by tavisca-dchougule** [comment]
@ytokas  modified code. please review.

**Comment by tavisca-dchougule** [comment]
@ytokas  exceptions handled, please review the code.

**Comment by tavisca-dchougule** [comment]
@ytokas  modified code after review...please review.

File: 01 Operator Overloading/ClassLibrary1/ExchangeRateProvider.cs [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using OperatorOverloading.Interfaces;
+using System.Configuration;
+using ClassLibrary1;
+
+namespace OperatorOverloading.dbl
```
Comment (ytokas) [inline 01 Operator Overloading/ClassLibrary1/ExchangeRateProvider.cs:11]: namespace naming convention incorrect, Assignment rejected

**Pull request merged**